README.md

```diff
@@ -217,13 +217,15 @@ Illegal Instruction error | Check if your CPU supports AVX while your OS does no
 
 ### OS X
 
-It is essential to use a 64-bit gfortran to compile Julia dependencies. The gfortran-4.7 (and newer) compilers in brew and MacPorts work for building Julia.
+It is essential to use a 64-bit gfortran to compile Julia dependencies. The gfortran-4.7 (and newer) compilers in brew compilers in Brew, Fink, and MacPorts work for building Julia.
```

Comment:
you can add another commit to the same pull request by modifying https://github.com/odysseus9672/julia/blob/patch-1/README.md